lib/assert.js

```diff
@@ -314,21 +320,29 @@ function setEquiv(a, b, strict, memo) {
   if (a.size !== b.size)
     return false;
 
+  // This is a set of the entries in b which have been consumed in our pairwise
+  // comparison. Initialized lazily so sets which only have value types can
+  // skip an extra allocation.
+  let bEntriesUsed = null;
```

Comment:
nit: I'd rather you name it something like `usedEntries`, since `bEntriesUsed` sounds like a boolean predicating a need to use `.entries()`